assets/scss/custom/_navbar.scss

```diff
@@ -1,10 +1,27 @@
 ﻿.navbar {
   margin-bottom: 0;
 
-  .row .navbar-brand {
+  .navbar-brand {
     height: 100%;
   }
 
+  .navbar-user-organization {
+    font-size: 18px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.11;
+    letter-spacing: 0.18px;
+    color: #ffffff;
+    border-left: 2px solid #f3b00a;
+    padding-left: 23px;
+    margin-left: 8px;
+
+    span {
+      vertical-align: middle;
+    }
+  }
+
```

Comment:
You should be able just display text in navbar without custom css: https://getbootstrap.com/docs/3.4/components/#navbar-text

Reply:
Ok. But I have to use custom CSS to show left-border.